browser-switch/src/main/java/com/braintreepayments/browserswitch/BrowserSwitchActivity.java

```diff
@@ -1,44 +1,27 @@
 package com.braintreepayments.browserswitch;
 
 import android.app.Activity;
-import android.net.Uri;
 import android.os.Bundle;
 
-import androidx.annotation.Nullable;
+import androidx.annotation.VisibleForTesting;
+import androidx.appcompat.app.AppCompatActivity;
 
 /**
  * <a href="https://developer.android.com/guide/topics/manifest/activity-element.html#lmode">singleTask</a>
  * Activity used to receive the response from a browser switch. This Activity contains no UI and
  * finishes during {@link Activity#onCreate(Bundle)}.
  */
-public class BrowserSwitchActivity extends Activity {
+public class BrowserSwitchActivity extends AppCompatActivity {
```

Comment:
- How come this class is extending from `AppCompatActivity` now?
- Is this a breaking change?
- Will this cause any issues for merchants who want to use `FragmentActivity`?

Reply:
Hey yeah, good point I think this can be reverted. AppCompatActivity is the recommended approach now, but BrowserSwitchActivity is headless, there's no associated UI. It mainly exists to capture the deep link from the browser when re-directing back into the app.